src/hooks/use-create-editor.ts

```diff
@@ -49,35 +51,51 @@ export const useCreateEditor = () => {
     locale: store.cslLocale,
     sectionCategories: store.sectionCategories,
     isViewingMode: store.isViewingMode,
+    getSnapshot: store.getSnapshot,
   }))
 
   const api = useApi()
+  const params = useParams()
```

Comment:
what I wrote about use params on submissionId also applies here. Thanks!

Reply:
as per IM, we can do it later, this PR becomes a bit too much and the code is ok to be merged as it is now.